### ISSUE TITLE
Add Project Groups

### ISSUE THREAD

**Issue by itadventurer** [refugeehackathon/helperchain#12]
Maybe we should add groups of projects such that people can subscribe to many different projects.
E.g. one could have projects like „DRK Dresden“, „Caritas Dresden“ etc…
and group them for example in „Helping in Dresden“
## Tasks

[ ] Create migrations
[ ] Add option to apply for certain categories

**Comment by imanjundi** [comment]
Yeah, I also thought this idea would be really helpful. Otherwise any new project might not have any subscribers for a while.
